src/main/java/com/iexec/worker/docker/DockerRegistryConfiguration.java

```diff
@@ -19,24 +19,40 @@
 import lombok.*;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 
 import javax.annotation.PostConstruct;
+import javax.validation.constraints.Positive;
 
+import java.time.Duration;
 import java.util.List;
 import java.util.Optional;
 import java.util.stream.Collectors;
 
 @Slf4j
 @Configuration
 @ConfigurationProperties(prefix = "docker")
+@Getter
+@Setter
 public class DockerRegistryConfiguration {
 
-    @Setter
-    @Getter
     private List<RegistryCredentials> registries;
 
+    /**
+     * Min pull timeout expressed in minutes
+     */
+    @Positive
```

Comment:
Are you sure `@Positive` works?

Reply:
https://docs.jboss.org/hibernate/validator/6.0/api/org/hibernate/validator/constraints/time/DurationMin.html looks available

Reply:
You're right, see https://github.com/iExecBlockchainComputing/iexec-worker/pull/442/commits/3e0e13cbe8f49aa026d01b3f87e101d9d91d2f40.